OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -287,21 +287,18 @@ public IEnumerable<IRenderable> Render(WorldRenderer wr)
 			var world = args.SourceActor.World;
 			if (!world.FogObscures(pos))
 			{
+				var palette = wr.Palette(info.Palette + (info.IsPlayerPalette ? args.SourceActor.Owner.InternalName : ""));
```

Comment:
This needs remain
```suggestion
				var paletteName = info.Palette;
				if (paletteName != null && info.IsPlayerPalette)
					paletteName += args.SourceActor.Owner.InternalName;

				var palette = wr.Palette(paletteName);
```
otherwise we're effectively reverting #19476!
(And we might want to adapt the other cases to use the same.)

Reply:
Aha! Yea missed that null check, will change.